test/Succeed/exec-tc/executables

```diff
@@ -0,0 +1 @@
+/home/amelia/default/Projects/agda/dist-newstyle/build/x86_64-linux/ghc-9.2.2/Agda-2.6.3/build/agda/agda
```

Comment:
This looks fishy.

Reply:
Ah, yeah. Accidentally did `git add -a` at some point 🤦‍♀️ There was an extra Nix file, too, but I hadn't seen this one.